uint8   ctrl_mode
int16   x_spd
int16   y_spd
int16   x_offset
int16   y_offset
float32 w_spd
---
int64 sum